app/javascript/components/widgets/twitter/tweet_body.jsx

```diff
@@ -41,15 +54,34 @@ OneMediaType.propTypes = {
   linkUrl: PropTypes.string.isRequired,
 };
 
+export const QuoteTweet = ({ displayText, linkUrl }) => (
+  <React.Fragment>
+    <TweetText text={displayText} />
+    <MediaWrapper>
+      <Microlink url={linkUrl} style={quoteStyle} size="small" />
+    </MediaWrapper>
+  </React.Fragment>
+);
+
+QuoteTweet.propTypes = {
+  displayText: PropTypes.string.isRequired,
+  linkUrl: PropTypes.string.isRequired,
+};
+
+// The 'RT' symbol given to retweets by twitter
```

Comment:
Oh ok. I think this may have helped clarify the preceding commit. I think `RetweetSymbol` would also be clearer.